packages/stylelint-plugin-ckeditor5-rules/lib/rules/license-header.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+// Leading comment in a file will be considered license, and thus eligible
+// for automatic fixing if it contains any of the strings below:
+const LICENSE_IDENTIFIERS = [
+	'@license',
+	'Copyright',
+	'copyright'
+];
+
+const stylelint = require( 'stylelint' );
+
+const { report, ruleMessages } = stylelint.utils;
+
+const ruleName = 'ckeditor5-plugin/license-header';
```

Comment:
```suggestion
const ruleName = 'ckeditor5-rules/license-header';
```

Let's rename `plugin` to `rules` as we have in `eslint`.